domains/dragan-leskovar

```diff
@@ -0,0 +1,11 @@
+{
+    "description": "Web developer portfolio",
+    "repo": "https://github.com/draganl/dragan-leskovar.io",
+    "owner": {
+        "username": "draganle",
+        "email": "[drleskov@inet.hr]"
+    },
+    "record": {
+        "CNAME": "draganl.github.io/dragan-leskovar.io"
```

Comment:
Invalid CNAME. CNAME cannot contain protocols, file paths, URL parameters, or anything combination of those. See docs for more information: https://docs.is-a.dev/ Reopen PR when this is fixed.